### PR TITLE
Tampere search channel changes for ol2 to ol4

### PR DESCRIPTION
**Pull request by ZenJappanen**
Ported ol2 functions to ol4 but this is only partial solution. There is a problem with the MapMoveRequest. Old implementation provided bounding box to the request and it seemed to work. The API has changed since and it requires zoomLevel (0-12) as attribute and not bounding box. 

This commit includes the request with bounding box and it does not zoom correctly. Is it possible to have old request functionality with bbox back?

### CODE REVIEW

**Comment by ZenJappanen** [comment]
New version with fixed zoom.

**Comment by ZenJappanen** [comment]
Reopened.

File: bundles/tampere/search-from-channels/instance.js [hunk 1]
```diff
@@ -806,16 +806,27 @@ Oskari.clazz.define(
...
+                text : {
+                  scale : 1.3,
+                  fill : {
+                    color : 'rgba(0,0,0,1)'
+                  },
+                  stroke : {
+                    color : 'rgba(255,255,255,0.8)',
+                    width : 2
+                  },
+                  labelProperty: 'test_property'
```
Comment (ZakarFin) [inline bundles/tampere/search-from-channels/instance.js]: Seems you don’t have an attribute named ’test_property’ for the features so this styling can be removed.

**Comment by ZakarFin** [comment]
Check the style if everything is needed there. At least labeling seems a bit off. Other than that this looks ready for merging 👍

**Pull request merged**